### DISCUSSION TITLE
개발 못하는 날 공지 & 기타 사항

### DISCUSSION THREAD

**Discussion by aploga** [JeonDohyeon/imitated-seed#5] [Announcements]
11/19 GoormIDE 서비스 점검으로 못합니다
- - -
저를 Collaborator로 추가하신 것은 함께 개발하자는 뜻으로 판단해 지금부터 이 repo에 개발하겠습니다.

**Comment by HotoRas** [comment]
코드 수정사항 고지 @aploga 
- aploga 브랜치에서 config.json을 삭제하고 git ignore 처리했습니다.
다음 git pull시 config.json을 미리 백업한 후 처리해주시면 됩니다.
  + 사유: 설마 저걸 그대로 냅둔 상태에서 배포하려고 하신 건 아니죠?

**Reply by aploga** [reply]
엌ㅋㅋ 감사합니당
